src/qt/locale/bitcoin_nl.ts

```diff
@@ -2665,27 +2665,27 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Using the fallbackfee can result in sending a transaction that will take several hours or days (or never) to confirm. Consider choosing your fee manually or wait until your have validated the complete chain.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Het gebruik van de terugvalvergoeding kan resulteren in het verzenden van een transactie die enkele uren of dagen (of nooit) nodig heeft om te bevestigen. Overweeg om uw tarief handmatig te kiezen of wacht tot u de volledige ketting heeft gevalideerd.</translation>
```

Comment:
`ketting` => `keten` to be more consistent with the other transalations of "chain"